templates/edit-meta.tmpl

```diff
@@ -255,7 +255,7 @@
 						</select>
 					</dd>
 					<dt><label for="rtl">Direction</label></dt>
-					<dd><input type="checkbox" id="rtl" name="rtl" {{if .Post.IsRTL.Bool}}checked="checked"{{end}} /><label for="rtl"> right-to-left</label></dd>
+					<dd><input type="checkbox" id="rtl" name="rtl" value="true" {{if .Post.IsRTL.Bool}}checked="checked"{{end}} /><label for="rtl"> right-to-left</label></dd>
```

Comment:
should this be false and unchecked by default?

Reply:
The value doesn't need to be `false`; if it's unchecked the element simply won't be included in POST, and we have code on the server side to handle that. The default value should reflect whatever's in the database, which this does. (It's only ever visible when a post has already been stored.)

(There is an argument to be made that we should allow these settings to be changed at the beginning of a post rather than after its first save, but that's not the state of the code at the moment.)

Reply:
There's also an argument to be made that this should be a select dropdown or a set of radio buttons, especially since the server-side code supports an "auto" setting (that changes RTL status based on the post language) that isn't available here. But I'm not sure if the "auto" code has been written, so I'm leaving that alone for now.

Reply:
ok, I'm just thinking out loud with your help :smile: 

So if the value doesn't get sent if the box is not checked, how do we change the stored value when unchecking the box and submitting?